api/src/main/java/org/openmrs/api/db/hibernate/HibernatePersonDAO.java

```diff
@@ -72,6 +73,121 @@ public void setSessionFactory(SessionFactory sessionFactory) {
 		this.sessionFactory = sessionFactory;
 	}
 	
+	/**
+	 * This method executes a Lucene search on persons based on the soundex filter with one search name given
+	 * 
+	 * @param name a person should match using soundex representation
+	 * @param birthyear the birthyear the searched person should have 
+	 * @param includeVoided true if voided person should be included 
+	 * @param gender of the person to search for 
+	 * @return the set of Persons that match the search criteria 
+	 */
+	private Set<Person> executeSoundexOnePersonNameQuery(String name, Integer birthyear, boolean includeVoided , String gender) {
+		// Use the Soundex Search Mode for each query
+		String matchMode = Context.getAdministrationService().getGlobalProperty(OpenmrsConstants.GLOBAL_PROPERTY_PATIENT_SEARCH_MATCH_MODE);
+		Context.getAdministrationService().setGlobalProperty(OpenmrsConstants.GLOBAL_PROPERTY_PATIENT_SEARCH_MATCH_MODE, OpenmrsConstants.GLOBAL_PROPERTY_PATIENT_SEARCH_MATCH_SOUNDEX);
```

Comment:
What happens when two people search for patients at the same time in different browsers?